### ISSUE TITLE
Not all Etcd3 class methods accessible

### ISSUE THREAD

**Issue by xpepermint** [mixer/etcd3#9]
I noticed that the `client.if()` method and some other methods are undefined. In the source code looks like it should work but the `npm` package doesn't provide it.

```
const etcd = new Etcd3();
console.log(etcd.if) // -> undefined
```

**Comment by connor4312** [comment]
Thanks for the issue! It doesn't look like the package has been published since the transaction builder was implemented. I'll take care of that today or tomorrow most likely.

**Comment by xpepermint** [comment]
Thx!

**Comment by xpepermint** [comment]
Hey, any progress?

**Comment by connor4312** [comment]
Sorry for the late update here, we've been busy with a rebranding and such. I plan to release the next version with password auth and role management here: https://github.com/mixer/etcd3/pull/11. This will bring etcd3 to near parity (barring namespaces) with the official/primary Go client.

**Comment by connor4312** [comment]
This has now been published in 0.2.0, You can read the full changelog here: https://github.com/mixer/etcd3/blob/master/changelog.md#020-2017-06-03

**Comment by xpepermint** [comment]
Thx!
